src/backend/gl/Cargo.toml

```diff
@@ -12,10 +12,15 @@ documentation = "https://docs.rs/gfx_device_gl"
 workspace = "../../.."
 
 [lib]
-name = "gfx_device_gl"
+name = "gfx_backend_gl"
+
+[features]
+default = ["glutin"]
+headless = ["glutin"]
```

Comment:
I think we can remove the `headless` feature as it got promoted to a first-class feature with the introduction of `Headless` trait.